modules/cluster-logging-elasticsearch-retention.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-elasticsearch.adoc
+
+[id="cluster-logging-elasticsearch-retention_{context}"]
+= Configuring Elasticsearch retention policy
+
+
+Each Elasticsearch index adds load and overhead to the Elasticsearch cluster. You can use the Elasticsearch rollover index API to automatically rollover indices when they reach a specified age. When an index reaches the designated age, Elasticsearch rolls over the index, deleting the current index and creating a new index. Rolling over and deleting indices optimizes the indices and helps Elasticsearch performance. 
+
+You specify how long Elasticsearch stores indices using a separate _retention policy_ for each of the three log sources, infrastucture logs, application logs, and audit logs, using the Cluster Logging Custom Resource (CR), which writes the policy to the Elasticsearch Custom Resource (CR).
+
+////
+Only maxAge in current implementation:
+You can specify any of the following conditions:
+* `max_age`. The maximum age that is allowed before triggering a rollover for the _hot_ indices, or those ​indices that are actively being updated and queried. Specify a value in days (d), hours (h), minutes (m), or seconds (s). 
+* `max_docs`. The maximum number of documents allowed in an index before triggering a rollover. Documents added since the last refresh are not included in the document count. The document count does not include documents in replica shards.
+* `max_size`. The maximum size the index can be before a rollover is triggered. Specify a value in bytes (b), kilobytes (kb), megabytes (mb), or gigabytes (gb). 
+////
+
+For example, the following retention policy rolls over active indices for the infrastructure logs every three days and the old indices are deleted seven days after rollover. {product-title} checks every 15 minutes to determine if the infrastructure indices need to be rolled over.
+
+----
+apiVersion: "logging.openshift.io/v1"
+kind: "Elasticsearch"
+metadata:
+  name: "elasticsearch"
+spec:
+  indexManagement:
+    policies: <1>
+    - name: infra-policy
+      pollInterval: 15m
+      phases:
+        hot:
+          actions:
+            rollover:
+              maxAge:   3d
+        delete:
+          minAge: 7d
+    mappings: <2>
+    - name:  node.infra 
+      policyRef: infra-policy
+      aliases:
+      - infra  
+...
+----
+<1> For each log source, the retention policy indicated when to delete and rollover logs for that source. The policy includes: 
+* When to rollover the indices, determine by `maxAge`. This setting comes from the Cluster Logging CR.
+* When to check if rollover should be performed, determined by `pollInterval`. This setting is the default.
+* When the rolled-over indicies will be deleted after the rollover, determined by `minAge`. This setting is the default.
```

Comment:
@jcantrill I assume the `pollInterval` and `minAge` parameters are defaulted and cannot be set without taking the EO inti unmanged state?